26-register_publication/543-creator_begins_wizard_registration_and_navigates_do_description_tab.feature

```diff
@@ -0,0 +1,22 @@
+  Feature: Creator begins Wizard registration and navigates to Description tab
+
+  Scenario: Creator begins Wizard registration and navigates to Description tab
+    Given Creator begins registering a Publication in the Wizard
+    When they navigate to the Description tab
+    Then they see the tab Description is selected

```

Comment:
```suggestion
    Then they see the Description tab is selected
```

Reply:
evt omvent. men synes det ser rart ut at vi først skriver "Description tab" og deretter "the tab Description"